practice/earthquake/site.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+/**
+ * Makes linter ignore ol variable
+ */
+if (!window) {
+  var ol = '';
+}
+
+var earth = new ol.layer.Tile({
+  source: new ol.source.OSM()
+});
+
+/**
+ * Requests the earthquake information for the last seven days from the
+ * earthquake USGS database. It returns this as a promise.
+ */
+function getEarthquakes() {
+  var url = 'http://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_week.geojson';
+  return Promise.resolve($.ajax({
+    dataType: 'json',
+    url: url,
+  }));
+}
+
+/**
+ * Takes a single earthquake object in as an argument, and returns a simpler
+ * earthquake containing only magnitude, age in hours, and an array of XY
+ * coordinates.
+ */
+function formatSingleEarthquake(Inearthquake) {
+  if (Inearthquake.properties.mag > 0) {
+    var effectiveMagnitude = Inearthquake.properties.mag;
+  } else {
+    effectiveMagnitude = .001;
```

Comment:
I believe not having a `var` on the second assignment works because of JS variable hoisting (which is confusing), but in general, write each branch like it can be copied and pasted and run independently.

```js
if (x > 0) {
  var y = 1;
} else {
  var y = -1;
}
```